.github/workflows/continuous-integration.yml

```diff
@@ -41,4 +41,7 @@ jobs:
 
       - name: Bazel Test All (opt)
         run: |
-          bazel test -c opt --noshow_progress --test_output=errors -- //xls/...
+          # Explicitly build the whole XLS except the benchmarking targets for the ZSTD module
```

Comment:
can you state why were are disabling the benchmarking rules w/ a comment? (is that because of #1455 ?)

Reply:
That is because we wanted to skip running long `place_and_route` flows in the main CI.

Reply:
maybe we could tag them as `manual`, see https://bazel.build/reference/be/common-definitions#common-attributes:~:text=it%27s%20executed%20remotely.-,manual,-keyword%20will%20exclude

Reply:
Tagged all benchmark macros and rules as `manual`. With this modification we were able to get back to the simpler bazel configuration with only one package and all tests and benchmarks defined in a single `BUILD` file. I've also reworked the CI workflows to take that change into account.

Unfortunately, there seems to be some kind of an issue with the propagation of the `tags` attribute for some of the XLS macros (see https://github.com/google/xls/issues/1487). I'm working on fixing that in https://github.com/google/xls/pull/1488